definitions/ontwikkel/discovery_dev_HuisartsVVTInzage.json

```
@@ -0,0 +1,127 @@
+{
+  "id": "dev:HuisartsVVTInzage",
+  "endpoint": "http://localhost:9091/discovery/dev:HuisartsVVTInzage",
+  "presentation_max_validity": 3600,
+  "presentation_definition": {
+    "id": "dev:HuisartsVVTInzage",
+    "format": {
+      "ldp_vc": {
+        "proof_type": [
+          "JsonWebSignature2020"
+        ]
+      },
+      "jwt_vc": {
+        "alg": [
+          "PS256"
+        ]
+      },
+      "ldp_vp": {
+        "proof_type": [
+          "JsonWebSignature2020"
+        ]
+      },
+      "jwt_vp": {
+        "alg": [
+          "ES256"
+        ]
+      }
+    },
+    "input_descriptors": [
+      {
+        "id": "id_uzicert_uracredential",
+        "name": "Care organization identity from CIBG-issued UZI-server certificate",
+        "purpose": "Finding a care organization for authorizing access to medical metadata.",
+        "constraints": {
+          "fields": [
+            {
+              "path": [
+                "$.vc.type",
+                "$.type"
+              ],
+              "filter": {
+                "type": "string",
+                "const": "X509Credential"
+              }
+            },
+            {
+              "path": [
+                "$.issuer"
+              ],
+              "purpose": "Whe can only accept credentials from a trusted issuer",
+              "filter": {
+                "type": "string",
+                "pattern": "^did:x509:0:sha256:GwlhBZuEFlSHXSRUXQuTs3_YpQxAahColwJJj35US1A[\\w\\-.:%]*$"
+              }
+            },
+            {
+              "id": "organization_name",
+              "path": [
+                "$.credentialSubject[0].subject.O"

```

I think we should add the non-indexed options here as well (and at all other places where credentialSubject[0] is accessed):

```suggestion
                "$.credentialSubject[0].subject.O",
                "$.credentialSubject.subject.O"

```